ophidian/ophidian

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env sh
+
+set -eu
+
+# This script involves just enough shell to bootstrap our way to a working
+# system where we don't have to write any more shell and everything can
+# be in Python. Its responsibilities are getting us a working version of
+# pyenv, getting us a working version of Python, and then getting out of
+# the way and running Python.
+
+# Write stdout to stderr, preserving the original stdout in fd 3
+exec 3<&1 1>&2
+
+ROOT="$(dirname $0)"
+REAL_MAIN="$ROOT/src/ophidian/__main__.py"
+
+
+if [ ! -e $REAL_MAIN ] ; then
+    echo "ophidian script is in the wrong directory (expected $REAL_MAIN to exist)" >&2
+    exit 1
+fi
+
+export OPHIDIAN_HOME="${OPHIDIAN_HOME:-"$HOME/.cache/ophidian-storage"}"
```

Comment:
Is something on the system (or a task before this point) guaranteed to create `${HOME}/.cache`? Or would the relevant Ophidian bits do that if necessary?